### ISSUE TITLE
SlevomatCodingStandard.Namespaces.FullyQualifiedClassNameInAnnotation.NonFullyQualifiedClassName false positive for never in a union

### ISSUE THREAD

**Issue by marekdedic** [slevomat/coding-standard#1414]
Hi,
I encountered this behaviour which I believe is incorrect. For this code snippet:

```php

namespace MyNamespace;

/**
 * @return int|never
 */
function weird(bool $a) {
    if ( $a ) {
        die;
    }
    return 0;
}
```

it produces the following error: `Class name \MyNamespace\never in @return should be referenced via a fully qualified name.` I believe this is incorrect and a false positive.

**Comment by kukulich** [comment]
What version do you use?

**Comment by marekdedic** [comment]
I'm on 8.3.0 and phpcs 3.7.1

**Comment by kukulich** [comment]
Fixed in https://github.com/slevomat/coding-standard/commit/e332564d6debc5390e809bc5d889c562c0822308

**Comment by marekdedic** [comment]
Great, thanks!
